README.md

```diff
@@ -1 +1,124 @@
-# wordpress-docker
\ No newline at end of file
+# WordPress Dockerized
+
+## Table of Contents
+
+1. [Introduction](#introduction)
+2. [Prerequisites](#prerequisites)
+3. [Quickstart](#quickstart)
+4. [Usage](#usage)
+5. [Environment Variables](#environment-variables)
+6. [Contact](#contact)
+7. [Checklist](checklist.pdf)
+
+---
+
+## Project Description
+
+This repository provides a minimal and functional Docker-based setup for running WordPress locally. It uses the official WordPress and MariaDB containers via Docker Compose. All content is persisted using volumes, and the setup is ready to be deployed via Docker with a custom (optional) `.env` configuration.
+
+---
+
+## Prerequisites
+
+```sh
+sudo apt update && sudo apt install -y docker.io
```

Comment:
In the best case you let the reader know docker is a prequisite with a small line of text, then add this codebox, but this is more of a suggestion for the future.

On the VM docker.io package is already pre-installed, but not with the compose plugins, which you need to install on your vm.